hack/lib/log/system.sh

```diff
@@ -143,7 +143,7 @@ function os::log::system::internal::plot() {
 
     local records
     local width
-    records="$(( $( cat "${datafile}" | wc -l ) - 1 ))" # one of these lines will be the header comment
+    records="$(( $( wc -l <"${datafile}" ) - 1 ))" # one of these lines will be the header comment
```

Comment:
We are not always consistent, but it looks like we usually put whitespace on either side of the redirect operator:
```bash
wc -l < "${datafile}"
```